doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,305 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
it would be great to add size limitations, defaults, min, max to fields where it make sense